include/port/async-pipe.h

```diff
@@ -25,6 +27,10 @@ int async_pipe_read(async_pipe_t pipe, void* msg, int len, async_pipe_onread cal
 // return: 0-success, other-error
 int async_pipe_write(async_pipe_t pipe, const void* msg, int len, async_pipe_onwrite callback, void* param);
 
+static void aio_onread(sigval_t sigval);
+
+static void aio_onwrite(sigval_t sigval);
```

Comment:
Declare static aio_onread/aio_onwrite ?

Reply:
Yes, compiler didn't found this functions in linux-async-pipe.c without declaration in this include file. Maybe this functions can be declared in linux-async-pipe.c?